mailpoet/prefixer/fix-doctrine.php

```diff
@@ -25,23 +25,6 @@
 $php7CachedItem = str_replace('final class CacheItem', 'final class TypedCacheItem', $php7CachedItem);
 file_put_contents(__DIR__ . "/../vendor-prefixed/doctrine/cache/lib/Doctrine/Common/Cache/Psr6/TypedCacheItem.php", $php7CachedItem);
 
-// Replace PHP8 syntax in ReflectionReadonlyProperty.
```

Comment:
Why is this code not necessary anymore?

Reply:
Thanks. My bad, because I misunderstood the comment. I thought it is not needed with the new version.
I reverted this change [here](https://github.com/mailpoet/mailpoet/pull/5641/commits/bb5c8b15cb8a9ca1f7b287a04f9328b71331653d).